frontend/src/components/headers/DocumentHeader.tsx

```diff
@@ -45,6 +72,23 @@ function DocumentHeader() {
 		navigate(`/${workspaceState.data?.slug}`);
 	};
 
+	const handleDocumentTitleChange = (
+		e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+	) => {
+		setDocumentTitle(e.target.value);
```

Comment:
It seems that it allows empty string.
With `useForm`, we can handle validation.